website/src/pages/404.tsx

```diff
@@ -1,15 +1,37 @@
+import { Box, Center, Link, Text, useColorModeValue } from "@chakra-ui/react";
 import Head from "next/head";
+import { useRouter } from "next/router";
+import { FiAlertTriangle } from "react-icons/fi";
+import { getTransparentHeaderLayout } from "src/components/Layout";
+
+function Error() {
+  const router = useRouter();
+  const backgroundColor = useColorModeValue("white", "gray.800");
+  const backgroundColor2 = useColorModeValue("gray.50", "gray.900");
 
-export default function Error() {
   return (
     <>
       <Head>
         <title>404 - Open Assistant</title>
         <meta name="404" content="Sorry, this page doesn't exist." />
       </Head>
-      <main className="flex h-3/4 items-center justify-center overflow-hidden subpixel-antialiased text-xl">
-        <p>Sorry, the page you are looking for does not exist.</p>
-      </main>
+      <Center bg={backgroundColor2} flexDirection="column" gap="4" fontSize="lg" className="subpixel-antialiased">
+        <Box bg={backgroundColor} p="10" borderRadius="xl" shadow="base">
+          <Box display="flex" flexDirection="column" alignItems="center" gap="8">
+            <FiAlertTriangle size="30" color="DarkOrange" />
+            <Box display="flex" flexDirection="column" alignItems="center" gap="3">
+              <Text>Sorry, the page you are looking for does not exist.</Text>
```

Comment:
Can you merge in my changes? I added in a button to file a bug here.

Reply:
working on it now